src/routes/meteors.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const asteroidsController = require('../controllers/asteroidsController');
+
+const router = express.Router();
+
+router
+    .route('/')
+    .get(asteroidsController.getAsteroids)
+    .post(asteroidsController.receivePictureFromRover);
```

Comment:
See comment above - we should return bicycle picture as e response to user data endpoint, asteroidsController doesn't fit here

Reply:
ah, I see, sure will move to userController